src/core/render-core.js

```diff
@@ -18,7 +18,6 @@ async function createBrowser(opts) {
   }
   browserOpts.headless = !config.DEBUG_MODE;
   browserOpts.args = ['--no-sandbox', '--disable-setuid-sandbox'];
-  browserOpts.executablePath = 'google-chrome';
```

Comment:
This seems to break local tests, even though the tests run on pipeline works. There seems to be an environment variable to control this option couple of code lines above this. So lets just set `google-chrome` in Heroku env vars in QA/prod. I understood this was done for reducing the Heroku slug size?

Reply:
Yup makes sense. (This is prob some linux/mac issue as this works fine for me)